glue/viewers/profile/state.py

```diff
@@ -230,6 +230,13 @@ def _reset_y_limits(self, *event):
                 if y_max > y_min:
                     self.y_min = y_min
                     self.y_max = y_max
+                elif np.allclose(y_min, y_max):
+                    if np.allclose(y_min, 0):
+                        self.y_min = -0.1
+                        self.y_max = 0.1
+                    else:
+                        self.y_min = y_min * 0.9
+                        self.y_max = y_max * 1.1
```

Comment:
```suggestion
                elif np.allclose(y_min, y_max):
                    if y_min == 0.0:
                        dy = np.finfo(y_min).resolution**2
                    else:
                        dy = abs(0.1 * y_min)
                    self.y_min = y_min - dy
                    self.y_max = y_max + dy
```
still somewhat arbitrary choice for "very small" limits, but I'd prefer something that still shows it is not all 1e-12 or similar.